pom.xml

```diff
@@ -68,7 +68,7 @@
       <groupId>${project.groupId}</groupId>
       <artifactId>jitsi-lgpl-dependencies</artifactId>
       <optional>true</optional>
-      <version>1.1-20190327.160813-5</version>
+      <version>1.1-0-gc27d819</version>
```

Comment:
I wonder if this is considered an older version. Should we bump to 1.2?

Reply:
https://github.com/jitsi/jitsi-lgpl-dependencies/pull/14